### DISCUSSION TITLE
Does Teleport follow any standards, like rfc6187, for doing certificate based authentication?

### DISCUSSION THREAD

**Discussion by alwaysastudent** [gravitational/teleport#10443] [Q&A]
Does Teleport support [rfc6187](https://datatracker.ietf.org/doc/html/rfc6187) or any other standards for implementing certificate based auth?

**Comment by webvictim** [comment]
Teleport doesn't currently use or support X509 certificates for SSH auth, although it does use them for all other protocols it supports.

SSH authentication is handled by SSH certificates as supported by OpenSSH.
